README.md

```diff
@@ -7,12 +7,28 @@
 [![Codecov](https://codecov.io/gh/cscherrer/Soss.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/cscherrer/Soss.jl)
 [![Coveralls](https://coveralls.io/repos/github/cscherrer/Soss.jl/badge.svg?branch=master)](https://coveralls.io/github/cscherrer/Soss.jl?branch=master)
 
-Soss is a library for _probabilistic programming_
+Soss is a library for _probabilistic programming_.
+
+## Getting started
+
+Soss is an officially registered package, so to add it to your project you can type
+```
+]add Soss
+```
+within the julia REPL and your are ready for `using Soss`. If it fails to precompile, it could be due to one of the following:
+
+* You have gotten an old version due to compatibility restrictions with your current environment.
+Should that happen, create a new folder for your Soss project, launch a julia session within, type
```

Comment:
Probably worth it to link to Pkg's instructions on this: https://julialang.github.io/Pkg.jl/stable/environments/#Creating-your-own-projects-1

Reply:
Right, I'll do that shortly.